checks/compile/haswell_fma_check.py

```diff
@@ -11,7 +11,7 @@
 class HaswellFmaCheck(rfm.CompileOnlyRegressionTest):
     def __init__(self):
         self.descr = 'check for avx2 instructions'
-        self.valid_systems = ['dom:login', 'daint:login']
+        self.valid_systems = []
```

Comment:
1/ we no longer have haswell systems
2/ this test will fail on alps:
```
Failed to find the following module(s): "craype-haswell"
```
do you want to update it (using procinfo) or delete the test ?